src/js/components/AddSpaceButton.tsx

```diff
@@ -21,7 +20,7 @@ const StyledAnchor = styled.a`
 
 export default function AddSpaceButton() {
   const dispatch = useDispatch()
-  const onClick = () => dispatch(resetTab())
```

Comment:
A bug with the "isOpen" arrow made be realize that this was overkill for what this button needs to do. It just needs to set current space to null.